src/types/my-resources/interfaces/myResources.interface.ts

```diff
@@ -8,3 +8,13 @@ export interface Lesson extends CommonEntityFields {
 export interface GetLessonsParams extends RequestParams {
   title: string
 }
+
+export interface Attachment extends CommonEntityFields {
+  fileName: string
+  title: string
+  size: number
```

Comment:
check which parameters attachment contains on the backend and add here, cause there are missed some of them

Reply:
fixed and refactored types in attachment